documentation/source/newsfragments/2045.bugfix.rst

```diff
@@ -0,0 +1 @@
+Tests skipped by default (created with `skip=True`) can again be ran manually by setting the :make:var:`TESTCASE` variable.
```

Comment:
```suggestion
Tests skipped by default (created with `skip=True`) can again be ran manually by setting the :envvar:`TESTCASE` variable.
```

It's not linking properly. The distinction gets me often.

Reply:
Fixed, thanks!